programs/mmm/src/util.rs

```diff
@@ -336,6 +351,10 @@ pub fn try_close_pool<'info>(pool: &Account<'info, Pool>, owner: AccountInfo<'in
         return Ok(());
     }
 
+    if pool.using_shared_escrow() && pool.shared_escrow_cap.unwrap_or(0) > pool.spot_price {
```

Comment:
is there any case that we want to close the pool here, perhaps when the buy limit has been reached and there are no nfts left in the pool?